exchange/websocket/buffer/buffer.go

```diff
@@ -2,13 +2,15 @@ package buffer
 
```

Comment:
I understand that this is an initial refactor, but it also seems odd that nothing in the manager or buffer responds to context.Done